pigpen-cascading/src/main/clojure/pigpen/cascading/core.clj

```diff
@@ -190,7 +196,7 @@
   [{:keys [id projections fields]} :- m/Project
    [{:keys [^Pipe pipe ancestor]}]
    _]
-  (let [context (pr-str `'{:ancestor ~(update-in ancestor [:opts] dissoc :tap)
+  (let [context (pr-str `'{:ancestor ~(clojure.walk/postwalk #(if (map? %) (dissoc % :tap) %) ancestor)
```

Comment:
I believe it's coming like that from the main oven, but I didn't trace it back further. Here is the code that produces it. You can't just run it directly since it uses some of our internal stuff, but it should give you an idea of what's going on.

```clojure
(defn compute-numbers [esf]
  (let [categories (pig/cogroup [(esf :on #(.get_synced_cookie %))]
                                (fn [key esfs] (let [ana-id (.get_ad_network_account_id key)
                                                     c (count esfs)]
                                                 [(str "ana-" ana-id) c]))
                                esf)
        groups (pig/cogroup [(categories :on identity)]
                            (fn [key group] (conj key (count group))))]
    groups))

(defn get-flow [input output]
  (->>
    (load-datastore input)
    (compute-numbers)
    (pig/store-tsv output)
    (cs/generate-flow (.getFlowConnector (CascadingHelper/get)))))
```